--- container/catalina/src/conf/catalina.policy	2009-07-24 21:36:46.000000000 +0200
+++ container/catalina/src/conf/catalina.policy.oden	2010-02-02 17:33:21.286218276 +0100
@@ -66,7 +66,7 @@
 };
 
 // These permissions apply to the commons-logging API
-grant codeBase "file:${catalina.home}/bin/commons-logging-api-1.1.1.jar" {
+grant codeBase "file:${catalina.home}/bin/commons-logging-api.jar" {
         permission java.security.AllPermission;
 };
 
--- container/catalina/etc/bootstrap.MF	2009-07-24 21:36:45.000000000 +0200
+++ container/catalina/etc/bootstrap.MF.oden	2010-02-02 17:33:21.286218276 +0100
@@ -1,5 +1,5 @@
 Manifest-Version: 1.0
 Main-Class: org.apache.catalina.startup.Bootstrap
-Class-Path: jmx.jar commons-daemon.jar commons-logging-api-1.1.1.jar tomcat-juli.jar
+Class-Path: jmx.jar commons-daemon.jar commons-logging-api.jar tomcat-juli.jar
 Specification-Title: Catalina
 Specification-Version: 1.0
\ No newline at end of file
