--- jasper/src/bin/jspc.sh.p8~	2013-09-07 11:43:35.333440178 +0200
+++ jasper/src/bin/jspc.sh	2013-09-07 11:45:22.179504056 +0200
@@ -21,32 +21,9 @@
 # $Id: jspc.sh 966908 2010-07-22 23:57:08Z rjung $
 # -----------------------------------------------------------------------------
 
-# resolve links - $0 may be a softlink
-PRG="$0"
-
-while [ -h "$PRG" ] ; do
-  ls=`ls -ld "$PRG"`
-  link=`expr "$ls" : '.*-> \(.*\)$'`
-  if expr "$link" : '.*/.*' > /dev/null; then
-    PRG="$link"
-  else
-    PRG=`dirname "$PRG"`/"$link"
-  fi
-done
- 
-PRGDIR=`dirname "$PRG"`
-EXECUTABLE=jasper.sh
-
-# Check that target executable exists
-if [ ! -x "$PRGDIR"/"$EXECUTABLE" ]; then
-  echo "Cannot find $PRGDIR/$EXECUTABLE"
-  echo "This file is needed to run this program"
-  exit 1
-fi
-
 if [ "$1" = "debug" ]; then
   shift
-  exec "$PRGDIR"/"$EXECUTABLE" debug "$@"
+  exec jasper5.sh debug "$@"
 else
-  exec "$PRGDIR"/"$EXECUTABLE" jspc "$@"
+  exec jasper5.sh jspc "$@"
 fi
