--- connectors/util/build.xml	2010-02-02 17:35:15.869280745 +0100
+++ connectors/util/build.xml.oden	2010-02-02 17:37:11.158343522 +0100
@@ -61,7 +61,9 @@
         <available property="commons-logging.present" file="${commons-logging.jar}"/>
         <available property="modeler.present" file="${commons-modeler.jar}"/>
 	<available property="jdk1.4.present" classname="java.lang.CharSequence" />
+        <condition property="build.javac.source" value="1.5" else="1.4">
         <available property="jdk1.5.present" classname="javax.net.ssl.CertPathTrustManagerParameters" />
+        </condition>
     </target>
 
     <target name="build-prepare" depends="detect">
@@ -89,6 +91,7 @@
 	       debug="${compile.debug}"
 	       optimize="off"
 	       verbose="off"
+               source="${build.javac.source}"
 	       excludes="**/CVS/**"
            encoding="ISO-8859-1">
             <classpath>
